report/10_System.md

```diff
@@ -1,5 +1,15 @@
 ## System's Perspective
 
-Short Introduction to System goes here.
+<!-- Short Introduction to System goes here.
+Double-check that for all the weekly tasks (those in the end of the lecture notes) you include the corresponding information. -->
 
-Double check that for all the weekly tasks (those in the end of the lecture notes) you include the corresponding information.
+Our web application itself is written in C#, using the ASP.NET web framework. 
+We have chosen this stack since it is the de-facto-standard for larger web applications (at least in Denmark), which makes it really well supported and maintained. This also means that many third-party packages is available to utilize, like Newtonsoft JSON, which is the de-facto way of (de)serialize JSON, and Prometheus-net for interacting with Prometheus.
```

Comment:
Shouldn't the introduction be more about saying what is in the system, and less about why we chose the things we did? It seems like the stuff that should go into separate parts.